packages/RNTester/js/components/RNTesterExampleList.js

```diff
@@ -39,14 +40,49 @@ type Props = {
   ...
 };
 
-class RowComponent extends React.PureComponent<{
+type ButtonState = {|active: boolean|};
+type ButtonProps = {
   item: Object,
   onNavigate: Function,
   onPress?: Function,
   onShowUnderlay?: Function,
   onHideUnderlay?: Function,
   ...
-}> {
+};
+
+function PlatformLogoContainer({platform}): React.Component {
+  return (
+    <View style={{flexDirection: 'row'}}>
+      {(!platform || platform === 'ios') && (
+        <Image
+          style={styles.platformLogoStyle}
+          source={require('../assets/apple.png')}
+        />
+      )}
+      {(!platform || platform === 'android') && (
+        <Image
+          style={styles.platformLogoStyle}
+          source={require('../assets/android.png')}
+        />
+      )}
+    </View>
+  );
+}
+
+class RowComponent extends React.PureComponent<ButtonProps, ButtonState> {
```

Comment:
Is there a specific reason for class components? You can also try using modern ES6 syntax https://veerasundar.com/blog/2019/03/rewriting-a-react-component-using-hooks-and-reactmemo/

Reply:
This choice was deliberate. The reason behind using class components was to adhere to the coding style for the rest of the app. It's awkward and frankly sometimes just ugly to mix class components and functional components - the rest of the RNTester app is written using class components. In fact, I made a mistake in the earlier part of my code when I used a functional component - I'll change this to a class component.

Reply:
@AnshG714 , there is no coherent coding style throughout the app. If you do a git history walkthrough of the app, all code when written is adhering to the coding pattern of the time. Functional components and hooks being the preferred pattern of choice in today's time, it would be wise to use that in the new code. Also, Functional components provide a cleaner code imo.

Reply:
I agree with the fact that functional components are better and the 'new' paradigm, and the code isn't 'coherent' but I have reservations about that for this PR. If we do choose to migrate to functional, I'll make those changes, but I don't think I want to mix it up just yet. Don't worry, if we migrate then I personally will change every component, but for now, do you really think only partially migrating the file to functional is the key to writing clean code? 

For now, at least for the duration of the time when I'm working on the main ListView, I want the coding style to be similar, it's easier for me to go through things very quickly and finish things up.

Again, don't worry, I'll take it upon myself to carry out the migration.